### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by markus-willems**

### CODE REVIEW

File: src/assets/styles/index.scss [hunk 1]
```diff
@@ -10,4 +10,18 @@ body {
...
-}
\ No newline at end of file
+}
+
+a {
+    color: $primary;
+    text-decoration: none;
+}
+
+h1,h2,h3,h4,h5,h6 {
```
Comment (felex0982) [inline src/assets/styles/index.scss]: can't find the spacebar bro?

File: src/views/Home.vue [hunk 2]
```diff
@@ -1,19 +1,82 @@
...
   name: 'Home',
   components: {
-    HelloWorld,
+    Card, Header, RegistrationForm, Navigation,
   },
 };
 </script>
+
+<style lang="scss">
+  .page-container {
```
Comment (felex0982) [inline src/views/Home.vue]: you could probably put all of this into index.scss

File: src/assets/styles/index.scss (hunk 1 above)
Comment (markus-willems) [reply src/assets/styles/index.scss]: Done :D

File: src/views/Home.vue (hunk 2 above)
Comment (markus-willems) [reply src/views/Home.vue]: Done!

**Comment by felex0982** [comment]
lgtm

**Pull request merged**